src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -948,14 +965,19 @@ def load_module(name, value):
                 loaded_sub_model = passed_class_obj[name]
             else:
                 # load sub model
+                sub_model_dtype = (
```

Comment:
Can't we just do
```
torch_dtype.get(name)
```

We'll just pass` None` down do `load_sub_model` if the submodel isn't in the map.  The subsequent call to `load_model_dict_into_meta` and sets the dtype to FP32
https://github.com/huggingface/diffusers/blob/3e35f56b00d73bc3c2d3bb69615176d0909fab8a/src/diffusers/models/model_loading_utils.py#L202